src/views/Error.vue

```diff
@@ -8,7 +8,7 @@
                 >
                 <p class="description">
                     Něco se nepovedlo. <br />
-                    Zkuste to, prosím znovu.
+                    Zkuste to prosím znovu.
```

Comment:
Máš pravdu. Ta čárka byla blbě. Správně česky je *Zkuste to, prosím, znovu*. Takže bych skoro nechal *Zkuste to znovu*.

Reply:
Tohle jsem už zkoumal a jsou správně obě verze, s čárkama i bez nich.

Reply:
https://proofreading.cz/cestina-pod-lupou/prosim-s-carkami-nebo-bez/

Reply:
nechal bych to tedy tak